lightning/src/ln/channelmanager.rs

```diff
@@ -1200,12 +1200,19 @@ pub enum PaymentSendFailure {
 	/// send_payment.
 	PathParameterError(Vec<Result<(), APIError>>),
 	/// All paths which were attempted failed to send, with no channel state change taking place.
-	/// You can freely retry the payment in full (though you probably want to do so over different
+	/// You can freely resend the payment in full (though you probably want to do so over different
```

Comment:
The above two variants also need this update. I would change them to mirror the copy in `AllFailedResendSafe`